Model/File/Reader/YamlReader.php

```diff
@@ -20,7 +20,7 @@ class YamlReader extends AbstractReader
      */
     public function parse($fileName)
     {
-        $content = SymfonyYaml::parse($fileName);
+        $content = SymfonyYaml::parse(file_get_contents($fileName));
```

Comment:
file_get_contents returns FALSE on error but SymfonyYaml::parse()  must be called w/ a string parameter.